src/main/java/org/beanpod/switchboard/entity/ChannelEntity.java

```diff
@@ -0,0 +1,34 @@
+package org.beanpod.switchboard.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Getter @Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity(name="Channel")
+@JsonIgnoreProperties({"hibernateLazyIntializer", "handler", "out", "in"})
+public class ChannelEntity {
+    @Id
+    String name;
```

Comment:
If we have `name` as the ID, does that mean no two channels will be able to have the same name, even if they're on different devices? if so, we can't use name. i feel like we might have to generate our own primary key.

Reply:
i think that is what @FGRCL told me too. @FGRCL already generates an id on his pull request, so we can use @FGRCL id as unique key